app/views/candidate_interface/new_references/reminder/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<% content_for :title, t('page_titles.references_send_reminder') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_offer_dashboard_reference_path(@reference), 'Back') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(
+      model: @reference,
+      url: candidate_interface_new_references_new_reminder_path(@reference),
+      method: :post,
+    ) do |f| %>
+      <h1 class="govuk-heading-l">
+        <%= t('page_titles.new_references_send_reminder', referee_name: @reference.name) %>
+      </h1>
+
+      <% if @reference.next_automated_chase_at.present? %>
+        <p class="govuk-body">
+          They‘ll also get an automatic reminder on <%= @reference.next_automated_chase_at.strftime('%-d %B %Y') %>.
```

Comment:
We don't use this date format (ie day full month name year) anywhere else from what I can gather, is it a conscious design change? Should we use the existing format? I've searched for this formatting and it's not in our codebase so it will be a bit anomalous. @frankieroberto ?

Reply:
If we're intending to use this format it would be better off named in https://github.com/DFE-Digital/apply-for-teacher-training/blob/98a06ead3753c6bad40554e22a8f77c64f34f4a7/config/initializers/date_formats.rb